### ISSUE TITLE
Issue with TuMangaOnline source

### ISSUE THREAD

**Issue by jcgram** [Skittyblock/aidoku-community-sources#410]
There’s an issue with the TuMangaOnline source, it says that you may be disconnected or there’s a problem with the source, when you go to the website there’s nothing wrong with it. Could you please fix it? This is for the Spanish speakers please.

**Comment by Skittyblock** [comment]
does this always happen for you? which manga? the source works fine for me

**Comment by jcgram** [comment]
> does this always happen for you? which manga? the source works fine for me

Hi! Ive been trying with different mangas and it only happens when it is a Manhwa, when doing it in the website it actually use other domain than the website domain at the moment you enter to a chapter, so maybe it crashes the app.

**Comment by Skittyblock** [comment]
if you could give a specific example of one that would be helpful

**Comment by jcgram** [comment]
Yeah, like “Jinx”, when I select the chapter it doesn’t let me read it. If you need the example of the website I can send it too. @Skittyblock 
![IMG_8262](https://github.com/Skittyblock/aidoku-community-sources/assets/140681976/9d89703f-7d52-4ce6-b260-b684dc976294)
![IMG_8261](https://github.com/Skittyblock/aidoku-community-sources/assets/140681976/79049b4c-d49d-4c9f-9890-51c469843a49)


> if you could give a specific example of one that would be helpful
